### PR TITLE
Support subscription initial position

### PR DESCRIPTION
**Pull request by nathluu**
Currently, NiFi pulsar consumer does not support the option to set subscription initial position. This PR is to add a new property to set the subscription initial position.

### CODE REVIEW

**Comment by nathluu** [comment]
Hi @david-streamlio,

Could you please help to review the PR?

Thank you!

**Comment by david-streamlio** [comment]
@nathluu Thanks for the contribution. I have a couple of suggestions / request.  

1. Can you separate the NiFi version update into a separate PR? I prefer to have one "change" per PR.
2. Can you add a default value for the subscription initial position of LATEST ?

Thanks

**Comment by nathluu** [comment]
Hi @david-streamlio,

I reverted the version update. The default value has been set to Latest already.
Please help to review again.

Thanks

File: nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/AbstractPulsarConsumerProcessor.java [hunk 1]
```diff
@@ -76,6 +70,11 @@
     static final AllowableValue FAIL = new AllowableValue(ConsumerCryptoFailureAction.FAIL.name(), "Fail",
             "Report a failure condition, and then route the message contents to the FAILED relationship.");
 
+    static final AllowableValue OFFSET_EARLIEST = new AllowableValue("Earliest", "Earliest",
+            "Set the offset to the earliest position");
```
Comment (david-streamlio) [inline nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/AbstractPulsarConsumerProcessor.java]: Please change the descriptions to match the enum definitions found in the API [docs](https://pulsar.apache.org/api/client/2.9.0-SNAPSHOT/org/apache/pulsar/client/api/SubscriptionInitialPosition.html#Earliest), 

E.g. "The earliest position which means the start consuming position will be the first message."  Just to clarify this to the NiFi users.  Thanks

File: nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/AbstractPulsarConsumerProcessor.java [hunk 2]
```diff
@@ -76,6 +70,11 @@
...
+    static final AllowableValue OFFSET_LATEST = new AllowableValue("Latest", "Latest",
+            "Set the offset to the latest position");
```
Comment (david-streamlio) [inline nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/AbstractPulsarConsumerProcessor.java]: Same comment as above, Make this "The latest position which means the start consuming position will be the last message."

**Review by david-streamlio — COMMENTED** [review]
Looks great. Just two minor documentation changes.

**Review by david-streamlio — APPROVED** [review]

**Comment by david-streamlio** [comment]
Added subscription initial position property

**Pull request merged**